src/vcpkg/commands.install.cpp

```diff
@@ -1105,6 +1105,7 @@ namespace vcpkg
             if (!maybe_manifest_scf)
             {
                 print_error_message(maybe_manifest_scf.error());
+                msg::println();
```

Comment:
Fixes
<img width="1276" alt="image" src="https://github.com/microsoft/vcpkg-tool/assets/41973254/defb6bf7-ab7e-4926-ac9d-543e48b6f2ad">

Reply:
FYI I repeated this in https://github.com/microsoft/vcpkg-tool/pull/1219 but https://github.com/microsoft/vcpkg-tool/pull/1203 resolves it by making `print_error_message` itself add the extra newline since I found other occurrences.